src/controllers/agent.js

```diff
@@ -0,0 +1,66 @@
+import dotenv from 'dotenv';
+import Model from '../../../Desktop/property-pro-api/src/models/model';
+
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+
+const agentModel = new Model('agents');
+dotenv.config();
+
+export const Agentsignup = async (req, res) => {
+  const {
+    firstName, lastName, password, email, phoneNumber
+  } = req.body;
+
+  const columns = 'first_name, last_name, password, email, phone_number ';
+  const strong = await bcrypt.hash(password, 12);
+  const values = `'${firstName}', '${lastName}', '${strong}', '${email}', '${phoneNumber}'`;
+  const checkEmail = await agentModel.select('*', ` where email = '${email}' `);
+
+  try {
+    if (checkEmail.rowCount < 1) {
+      const data = await agentModel.insertWithReturn(columns, values);
+      const newUser = { firstName, lastName, email };
+
+      const token = jwt.sign(
+        { ...newUser, id: data.rows.id },
+        process.env.TOKEN_KEY,
+        { expiresIn: '1hr' }
+      );
+      res
+        .status(201)
+        .send({
+          user: newUser,
+          token,
+          message: 'Account created successfully',
+        });
+    }
+  } catch (err) {
+    res.status(400).json({ message: err.stack });
+  }
```

Comment:
At line 31 you are using **res.send** to send back a response and also using **res.json**
to send back the other one..It's really a great idea but i will suggest you use either of the two